pjlib/src/pj/ssl_sock_ossl.c

```diff
@@ -2357,6 +2357,7 @@ static pj_status_t ssl_do_handshake(pj_ssl_sock_t *ssock)
 	    
 	    sess = SSL_get_session(ossock->ossl_ssl);
 
+#if OPENSSL_VERSION_NUMBER >= 0x1010100fL
 	    PJ_LOG(5, (THIS_FILE, "Session info: reused=%d, resumable=%d, "
 		       "timeout=%d",
 		       SSL_session_reused(ossock->ossl_ssl),
```

Comment:
shouldn't we remove the #if below? It becomes redundant now.

Reply:
Indeed it is redundant and not compilable. Changed in next commit.